taskcluster/ci/clean-corpus/kind.yml

```diff
@@ -58,10 +74,15 @@ tasks:
     
         run:
             using: run-task
+            # We're going to migrate to OpusCleaner in the long term so we can use a simple 'if' in the command
+            # for now without introducing an extra step and complexity
             command:
                 - bash
                 - -c
-                - $VCS_PATH/pipeline/clean/clean-corpus.sh $MOZ_FETCHES_DIR/{dataset_sanitized} /builds/worker/artifacts/{dataset_sanitized} auto {dataset}
+                - >-
+                    pip install -r $VCS_PATH/pipeline/clean/requirements/clean.txt &&
+                    if [ ${USE_OPUSCLEANER} = "true" ]; then dir="clean/opuscleaner"; else dir="clean"; fi &&
+                    $VCS_PATH/pipeline/${dir}/clean-corpus.sh $MOZ_FETCHES_DIR/{dataset_sanitized} /builds/worker/artifacts/{dataset_sanitized} auto {dataset}
```

Comment:
Thought: This is another example where the bash script vs python argparse makes it difficult to understand how everything is wired together. With named arguments this would be much easier to understand.